pkg/cmd/cli/cmd/rollout/rollout.go

```diff
@@ -134,3 +135,20 @@ func NewCmdRolloutUndo(fullName string, f *clientcmd.Factory, out io.Writer) *co
 	cmd.Example = fmt.Sprintf(rolloutUndoExample, fullName)
 	return cmd
 }
+
+const (
+	rolloutStatusLong = `
+Watch the status of the latest rollout, until it's done.`
```

Comment:
Meaning the command waits and blocks and only exit when the rollout is completed? In such case I'd vote for `oc rollout watch` or `oc rollout status --watch` (exits right away without the flag) for consistency.